src/clusterfuzz/_internal/bot/tasks/utasks/minimize_task.py

```diff
@@ -956,22 +1002,6 @@ def check_deadline_exceeded_and_store_partial_minimized_testcase(
   return time.time() > deadline
 
 
-def handle_minimize_deadline_exceeded(output: uworker_msg_pb2.Output):
```

Comment:
Are these functions just moved and not otherwise changed?

Reply:
Yes! I just needed to define them above `_ERROR_HANDLER`.